recipes/adagio/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
This file is already included in the tarball on PyPI, so please remove from this PR.

Reply:
Thanks!